lib/pages/slider_selector.dart

```diff
@@ -0,0 +1,298 @@
+                    });
+                  },
+                ),
+              ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  // -----------------------------------------------
+  // Creates a list of RangeSliders, based on their
+  // definition and SliderTheme customizations
+  // -----------------------------------------------
+  List<Widget> _buildRangeSliders() {
+    List<Widget> children = <Widget>[];
+    for (int index = 0; index < rangeSliders.length; index++) {
+      children
+          .add(rangeSliders[index].build(context, (double lower, double upper) {
+        // adapt the RangeSlider lowerValue and upperValue
+        setState(() {
+          rangeSliders[index].lowerValue = lower;
+          rangeSliders[index].upperValue = upper;
+        });
+      }));
+      // Add an extra padding at the bottom of each RangeSlider
+      children.add(SizedBox(height: 8.0));
+    }
+
+    return children;
+  }
+
+  // -------------------------------------------------
+  // Creates a list of RangeSlider definitions
+  // -------------------------------------------------
+  List<RangeSliderData> _rangeSliderDefinitions() {
+    return <RangeSliderData>[
+      RangeSliderData(
+          min: 0.0, max: 100.0, lowerValue: 10.0, upperValue: 100.0),
+      RangeSliderData(
+          min: 0.0,
+          max: 100.0,
+          lowerValue: 25.0,
+          upperValue: 75.0,
+          divisions: 20,
+          overlayColor: Colors.red[100]),
+      RangeSliderData(
+          min: 0.0,
+          max: 100.0,
+          lowerValue: 10.0,
+          upperValue: 30.0,
+          showValueIndicator: false,
+          valueIndicatorMaxDecimals: 0),
+      RangeSliderData(
+          min: 0.0,
+          max: 100.0,
+          lowerValue: 10.0,
+          upperValue: 30.0,
+          showValueIndicator: true,
+          valueIndicatorMaxDecimals: 0,
+          activeTrackColor: Colors.red,
+          inactiveTrackColor: Colors.red[50],
+          valueIndicatorColor: Colors.green),
+      RangeSliderData(
+          min: 0.0,
+          max: 100.0,
+          lowerValue: 25.0,
+          upperValue: 75.0,
+          divisions: 20,
+          thumbColor: Colors.grey,
+          valueIndicatorColor: Colors.grey),
+    ];
+  }
+}
+
+// ---------------------------------------------------
+// Helper class aimed at simplifying the way to
+// automate the creation of a series of RangeSliders,
+// based on various parameters
+//
+// This class is to be used to demonstrate the appearance
+// customization of the RangeSliders
+// ---------------------------------------------------
+class RangeSliderData {
+  double min;
+  double max;
+  double lowerValue;
+  double upperValue;
+  int divisions;
+  bool showValueIndicator;
+  int valueIndicatorMaxDecimals;
+  bool forceValueIndicator;
+  Color overlayColor;
+  Color activeTrackColor;
+  Color inactiveTrackColor;
+  Color thumbColor;
+  Color valueIndicatorColor;
+  Color activeTickMarkColor;
```

Comment:
Why do you need those Colors when you have declared them below?

Reply:
deleted file